target/linux/ath79/dts/ar1022_sitecom_wlr-7100.dts

```diff
@@ -147,7 +147,7 @@
 				read-only;
 			};
 
-			art: partition@7f0000 {
+			partition@7f0000 {
```

Comment:
Hi! Please move these changes to a different commit.

Reply:
hmm? Do you mean the standalone art: removals?

Reply:
Yes. This file does not contain nvmem changes, so it should be a separate commit.

Reply:
Done.